in_toto/runlib.go

```diff
@@ -66,7 +72,10 @@ the following format:
 If recording an artifact fails the first return value is nil and the second
 return value is the error.
 */
-func RecordArtifacts(paths []string) (map[string]interface{}, error) {
+func RecordArtifacts(paths []string, depth uint) (map[string]interface{}, error) {
```

Comment:
Oh yes, this makes sense. Totally forgot about this, thanks.

Reply:
This should be solved.. I just wonder if we can set a default value for parameters like in Python:
`def RecordArtifacts(paths, depth=0):`

I will google a bit and see if this is manageable, because otherwise I am afraid that always calling this function with depth=0 is very error prone. People really need to make sure to read the docs.

Reply:
I fear go does not have that feature. But I wouldn't worry too much about usage mistakes. If you pass a parameter to a function, you usually want to know what it is for, right? Also, although we export this function as public API, its mostly for internal use by `InTotoRun`.